raml-headers-form.html

```diff
@@ -197,7 +223,7 @@
                     <label>Header name</label>
                     <input
                       is="anypoint-input"
-                      class="name-change-input"
+                      class="name-change-input anypoint-text-input"
```

Comment:
So how's this different from current version in anypoint branch?
![image](https://cloud.githubusercontent.com/assets/3320020/26493783/732b76fc-4213-11e7-86c4-bed968a2dd94.png)